modules/reactor/src/meshgenerators/HexagonMeshTrimmer.C

```diff
@@ -0,0 +1,574 @@
+      if ((*elem_it)->neighbor_ptr(i) != nullptr)
+        if ((*((*elem_it)->neighbor_ptr(i))).subdomain_id() == block_id_to_remove)
+        {
+          node_list.push_back((*elem_it)->side_ptr(i)->node_ptr(0)->id());
+          node_list.push_back((*elem_it)->side_ptr(i)->node_ptr(1)->id());
+          boundary_info.add_side(*elem_it, i, external_boundary_id);
+          boundary_info.add_side(*elem_it, i, trimming_section_boundary_id);
+        }
+    }
+  }
+  const auto unique_it = std::unique(node_list.begin(), node_list.end());
+  node_list.resize(std::distance(node_list.begin(), unique_it));
+  // Mark those nodes that are on a boundary that requires conformality
+  // If both nodes of a side are involved, we should only move one node
+  std::vector<bool> node_list_flag(node_list.size(), false);
+  std::vector<Point> node_list_point(node_list.size(), Point(0.0, 0.0, 0.0));
+  for (unsigned int i = 0; i < slc_bdry_side_list.size(); i++)
+  {
+    dof_id_type side_id_0 = mesh.elem_ptr(std::get<0>(slc_bdry_side_list[i]))
+                                ->side_ptr(std::get<1>(slc_bdry_side_list[i]))
+                                ->node_ptr(0)
+                                ->id();
+    dof_id_type side_id_1 = mesh.elem_ptr(std::get<0>(slc_bdry_side_list[i]))
+                                ->side_ptr(std::get<1>(slc_bdry_side_list[i]))
+                                ->node_ptr(1)
+                                ->id();
+    // True means the bdry node is in the node list of the trimming interface
+    bool side_id_0_in =
+        !(std::find(node_list.begin(), node_list.end(), side_id_0) == node_list.end());
+    bool side_id_1_in =
+        !(std::find(node_list.begin(), node_list.end(), side_id_1) == node_list.end());
+
+    // True means on the removal side
+    bool side_node_0_remove = lineSideDeterminator((*mesh.node_ptr(side_id_0))(0),
+                                                   (*mesh.node_ptr(side_id_0))(1),
+                                                   bdry_pars[0],
+                                                   bdry_pars[1],
+                                                   bdry_pars[2],
+                                                   side_to_remove);
+    bool side_node_1_remove = lineSideDeterminator((*mesh.node_ptr(side_id_1))(0),
+                                                   (*mesh.node_ptr(side_id_1))(1),
+                                                   bdry_pars[0],
+                                                   bdry_pars[1],
+                                                   bdry_pars[2],
+                                                   side_to_remove);
+    // If both nodes of that side are involved in the trimming interface
+    if (side_id_0_in && side_id_1_in)
+      // The side needs to be removed from the sideset
+      // The other node will be handled by other element's side
+      boundary_info.remove_side(mesh.elem_ptr(std::get<0>(slc_bdry_side_list[i])),
```

Comment:
this isnt covered. Any meshes in the reactor module which could trigger this one?
You d need less than 12th symmetry right?

Reply:
Yes, same as the previous comment.